modules/ossm-config-individual-control-plane-infrastructure-node-cli.adoc

```diff
@@ -3,12 +3,12 @@
 // * service_mesh/v2x/ossm-deployment-models.adoc
 
 :_content-type: PROCEDURE
-[id="ossm-config-individual-control-plane-infrastructure-node_{context}"]
-= Configuring individual {SMProductShortName} control plane components to run on infrastructure nodes
+[id="ossm-config-individual-control-plane-infrastructure-node-cli_{context}"]
+= Configuring individual control plane components to run on infrastructure nodes using the CLI
 
-This task should only be performed if individual {SMProductShortName} control plane components (such as Istiod, the Ingress Gateway, and the Egress Gateway) will run on infrastructure nodes. 
+Perform this task if individual {SMProductShortName} control plane components, such as Istiod, the Ingress Gateway, and the Egress Gateway, run on infrastructure nodes. 
 
-If the control plane will run on a worker node, skip this task.
+If the control plane runs on a worker node, skip this task.
```

Comment:
Same comment as above (this task is about moving from worker nodes to infra nodes).

Reply:
Done